homeassistant/const.py

```diff
@@ -1031,6 +1031,29 @@ class UnitOfDataRate(StrEnum):
 """Deprecated: please use UnitOfDataRate.GIBIBYTES_PER_SECOND"""
 
 
+# Radiation units
+class UnitOfRadiation(StrEnum):
```

Comment:
I couldn't find an architectural discussion on this. These seem like a random collection of radiation units, but where are those used?

At this point, it seems like this PR is introducing dead code.

Reply:
They might not be used by HA Core at this point, but could be in the future and it would also add possibility to be used by custom integrations.

Reply:
Right, but that is not a reason to add things. As said, this is introducing dead code, which we don't allow. Custom integration do not need to function, therefore we require use in our Core code base.